### PR TITLE
Include some copy on the log in page to explain the process

### PR DESCRIPTION
**Pull request by mstriemer**
screenshot 2016-08-09 11 08 13

Fixes mozilla/addons#180.

### CODE REVIEW

File: src/olympia/users/templates/users/login.html [hunk 1]
```diff
@@ -15,7 +15,13 @@
   {% else %}
     {% include "users/login_help.html" %}
     <section class="island hero primary grid prettyform">
-      <h1>{{ _('Enter your email') }}</h1>
+      <h1>{{ _('Enter your email to log in or register') }}</h1>
+      <p class="island-note">
+        {% trans %}
+          As the Firefox Accounts migration continues we will detect if you
```
Comment (tofumatt) [inline src/olympia/users/templates/users/login.html]: A user might not realise it has ever started. Also "log in or register on this site or on Firefox Accounts" is hard to parse. Maybe:

"We are migrating existing Add-ons users to Firefox accounts. We will detect if your account needs migrating automatically."

Comment (mstriemer) [inline src/olympia/users/templates/users/login.html]: I was hoping "continues" would make it clear but that's definitely subtle. I like this.

**Comment by tofumatt** [comment]
r+

**Comment by mstriemer** [comment]
Heading change moved to https://github.com/mozilla/addons-server/pull/3239.
